src/figvalidationservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/FigValidationServiceAppTests.java

```diff
@@ -54,5 +54,24 @@ void locateMatchingApplicantsHttpResponseTest() throws Exception {
 		        String.class)
 		).contains("<respCode>1</respCode>");	
 	}
+	
+
+	/**
+	 * ValidateApplicantServiceHttpResponseTest - Basic HTTP test of the ValidateApplicantService validation operation endpoint.
+	 * 
+	 * This test is NOT expected to fetch and test the response against any real data, rather we just want an HTTP response from the 
+	 * operation indicating it's working. 
+	 * 
+	 * TODO - This will require recoding once ORDS has been connected. 
+	 * 
+	 * @throws Exception
+	 */
+	@Test
+	void validateApplicantServiceHttpResponseTest() throws Exception {
+		String request = "/validateApplicantService?orgPartyId=abcdef99901&applPartyId=def9908";
+		assertThat(this.restTemplate.getForObject("http://localhost:" + port + "/figvalidationservice" + request,
```

Comment:
I see the request string defined, but the port has me confused...

Is there ever a case where this is null or not defined? Sorry if this is dumb.

Reply:
Not that I am aware. 'Port' is a Junit supplied annotation that automatically assigns the designated port (in our case 8081) at the time the test is run. If it is equated to null, this would mean that the entire spring boot application failed to start for some reason.